src/app/store/stripFormMeta.ts

```diff
@@ -0,0 +1,9 @@
+import * as R from 'ramda';
+import { AppAction } from '../actions';
+
+const stripFormMeta = R.compose(
```

Comment:
Nice …but you don’t use is ?

Reply:
That was my first attempt but this is a bit limited and this is causing other issue, since we're expecting these metas to be defined in other sagas.
Serialization is a more radical solution for other potential issues and keep action consistent across content/background. I thought it would also fix the empty error message but unfortunately it doesn't. 
>  ;) see https://github.com/dis-moi/extension/pull/800#discussion_r585066876